pint_server/models.py

```diff
@@ -77,6 +77,7 @@ class ProviderServerBase(PintBase):
     # column type accordingly.
     ip = Column(postgresql.INET, primary_key=True)
     region = Column(String(100), primary_key=True)
+    ipv6 = Column(postgresql.INET)
```

Comment:
We've agreed on putting a unique index on ip and ipv6 right?

Reply:
No, last thing we agree (on the meeting yesterday) was an ID column as PK, which it is already done in another PR

Reply:
at least, not in the meeting but it makes sense to have it

Reply:
we can't have a unique index, at least, on `ip` because it is not unique, it would probably be the same for `ipv6`